src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.cpp

```diff
@@ -45,5 +45,65 @@ Value getTosaConstTensorSingleF32(PatternRewriter &rewriter, Operation *op,
   auto constOp = rewriter.create<ConstOp>(op->getLoc(), constType, constAttr);
   return constOp.getResult();
 }
+
+// Templated function to create a constant op for given type and shape.
+// T: storage C type.
+// Default template creates a constant tensor in T.
+template <typename T>
+llvm::Optional<Value> getConstTensor(PatternRewriter &rewriter, Operation *op,
```

Comment:
`Value` already had nullptr-like semantics - the `Optional` is overkill. But I guess this comes from upstream?

Reply:
Yes, this comes from upstream